store/entity-store.js

```diff
@@ -126,6 +127,9 @@ window.D2L.Siren.EntityStore = {
 		if (!entityId) {
 			return Promise.reject(new Error('Cannot fetch undefined entityId'));
 		}
+		if (!window.D2L.Siren.WhitelistBehavior.isWhitelisted(entityId)) {
+			return Promise.reject(new Error('Invalid request url; must be a valid whitelisted domain.'));
```

Comment:
This isn't the approach we've taken generally. It might be fine, but it also might not be. Other libraries we've made have opted to just not add the `Authorization` header, as opposed to making the library only work at all for d2l urls.

Reply:
@pbrown-d2l @awikkerink Are we sure this is the approach we want to release?

Reply:
This was sort of a breaking change for my repo. Tests began failing. I'm guessing the recommended way is to setup test mode during setup? Let me know if there is a different suggestion. Related PR is https://github.com/BrightspaceHypermediaComponents/activities/pull/243

Reply:
This has broken a lot of demos and tests :/ on most of the BHC org.

Reply:
If you are making calls to sites not on the whitelist in your tests and demos it is very easy to switch it off as Miles mentioned. Luckily, now your users tokens aren't at risk,